ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/css/radiobuttonview.css

```diff
@@ -0,0 +1,9 @@
+.cmp-adaptiveform-radiobutton { }
+.cmp-adaptiveform-radiobutton__widget { }
+.cmp-adaptiveform-radiobutton__label { }
+.cmp-adaptiveform-radiobutton__longdescription { }
+.cmp-adaptiveform-radiobutton__shortdescription { }
+.cmp-adaptiveform-radiobutton__questionmark { }
+[data-cmp-adaptiveform-radiobutton-hidden="true"] {
```

Comment:
I think this has been changed by @sakshi-arora1. Can you check the latest code

Reply:
done